src/lib/features/listening-history/model/specs/store.spec.ts

```diff
@@ -0,0 +1,19 @@
+import { addToListeningHistory, listeningHistory } from '../store';
+import { get } from 'svelte/store';
+import type { Episode } from '$lib/shared/api';
+
+const exampleEpisode: Episode = {
+  id: 1,
+  episodeNumber: 1,
+  podcastID: 1,
+  title: 'Episode 1',
+  audioUrl: 'http://example.com',
+  duration: 5,
+};
+
+const expectedHistory: number[] = [exampleEpisode.id];
+
+it('listen to an episode', () => {
+  addToListeningHistory(exampleEpisode);
+  expect(get(listeningHistory)).toEqual(expectedHistory);
```

Comment:
issue: this test doesn't test the network interactions, which I think are an important aspect

Reply:
Added podcast fetching tests in `src\lib\entities\episode\api\specs\get-episode.spec.ts`. Let me know if that's not sufficient.

Reply:
It's sufficient for testing `getEpisode`, but in this thread I was talking about the `addToListeningHistory` function, namely, its logic of communicating with the backend.

Reply:
Added more tests at `src\lib\features\listening-history\api` and `src\lib\features\like-episode\api`.